src/app/containers/CpsRelatedContent/index.test.jsx

```diff
@@ -63,4 +66,35 @@ describe('CpsRelatedContent', () => {
     renderRelatedContent();
     expect(document.querySelectorAll(`[role='complementary']`).length).toBe(1);
   });
+
+  it('should render timestamps in milliseconds when page data has timestamps in seconds', async () => {
+    const initialPromo = [
+      {
+        ...promos[0],
+        timestamp: 1234567890,
+      },
+    ];
+
+    const pageData = await preprocessor(
+      {
+        ...pidginPageData,
+        relatedContent: { groups: [{ promos: initialPromo }] },
+      },
+      cpsAssetPreprocessorRules,
+    );
+
+    const transformedPromos = path(
+      ['relatedContent', 'groups', 0, 'promos'],
+      pageData,
+    );
+
+    renderRelatedContent({
+      content: transformedPromos,
+    });
+
+    const renderedTime = document
+      .querySelector(`aside time`)
+      .getAttribute('datetime');
+    expect(renderedTime).toBe('2009-02-14');
```

Comment:
Does 1234567890/1234567890000 not convert to Friday, 13 February 2009 23:31:30?

Reply:
Umm, good point - the test passes, so maybe a timezone or daylight savings thing?

Need to investigate a bit more - don't want a test that fails depending on what date it is run 😬

Reply:
A job for tomorrow me thinks 🙂

Reply:
not sure if this is related but we had a similar issue with  the article timestamp tests failing on ci at odd hours in the morning, the [fix](https://github.com/bbc/simorgh/pull/4703/files) was to stub `Date.now` to force it to use the exact time we wanted

Reply:
👍 Updated test to just check the month and year - not the date